src/lang/english.txt

```diff
@@ -3774,6 +3776,7 @@ STR_COMPANY_VIEW_PRESIDENT_S_NAME_QUERY_CAPTION                 :Manager's Name
 STR_COMPANY_VIEW_GIVE_MONEY_QUERY_CAPTION                       :Enter the amount of money you want to give
 
 STR_BUY_COMPANY_MESSAGE                                         :{WHITE}We are looking for a transport company to take-over our company.{}{}Do you want to purchase {COMPANY} for {CURRENCY_LONG}?
+STR_BUY_COMPANY_HOSTILE_TAKEOVER                                :{WHITE}You are about to do a hostile takeover of {COMPANY}.{}{}You will get all their assets, pay off their loan and two years of profit. The total comes down to {CURRENCY_LONG} currently.{}{}Do you want to continue the hostile takeover?
```

Comment:
From the message:
> You will get all their assets, pay off their loan and two years of profit.

That might be interpreted as receiving two years of profit, but the actual cost includes two years of profit. Therefore, another way that part could be worded is:
> You will get all their assets in exchange for paying off their loan plus two years of profit.

Reply:
I really appreciate these comments, tnx :) Also cleaned up the next sentence a bit. Much better now!